e2e/cypress/support/api_commands.js

```diff
@@ -46,10 +46,14 @@ Cypress.Commands.add('apiLogout', () => {
         log: false,
     });
 
+    // Ensure we clear out these specific cookies
     ['MMAUTHTOKEN', 'MMUSERID', 'MMCSRF'].forEach((cookie) => {
         cy.clearCookie(cookie);
     });
 
+    // Clear remainder of cookies
+    cy.clearCookies();
```

Comment:
Seems weird to me and looks like a bug in Cypress.  By looking at it, it made the individual clearing of cookie redundant.  But when ran, it's definitely necessary.

Reply:
Right? I'm thinking it's because it's actually async, so it's reliant on timing. This doesn't feel "ideal" to me, but it should be pretty safe and stable?

Reply:
Yup, I'm good with it.